src/corelibs/U2Gui/src/util/DownloadRemoteFileDialog.cpp

```diff
@@ -207,21 +208,17 @@ void DownloadRemoteFileDialog::accept() {
             addToProject = false;
         }
     }
-    bool hasLoadOnlyDocuments = false;
-    foreach (const QString& resId, resIds) {
-        LoadRemoteDocumentMode mode = LoadRemoteDocumentMode_LoadOnly;
-        if (addToProject) {
-            mode = taskCount < OpenViewTask::MAX_DOC_NUMBER_TO_OPEN_VIEWS ? LoadRemoteDocumentMode_OpenView : LoadRemoteDocumentMode_AddToProject;
+    for (int i = 0; i < resIds.size(); i++) {
+        if (!addToProject) {
+            hints.insert(SHOW_REPORT_HINT, true);
```

Comment:
This should be a single report for the whole op, but not a spam of potentially 1000 notifications per each downloaded document.

Reply:
Yes, of course, it's how it works now!

Reply:
Ok, I see MultiTask handles it.